frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetailsActions.tsx

```diff
@@ -51,41 +53,45 @@ const PipelineDetailsActions: React.FC<PipelineDetailsActionsProps> = ({
           <DropdownItem key="upload-version" onClick={() => setIsVersionImportModalOpen(true)}>
             Upload new version
           </DropdownItem>,
-          <DropdownSeparator key="separator-create" />,
```

Comment:
in some places you disable the create button and others like this area you hide? Can we make it consistent and make them disabled instead

Reply:
I hide them instead of disabled, because that was done before in this component. But we can disable that also, let me change this.

Reply:
we hid them becuase a valid reason. we should disable so we can tell them why